docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM socrata/runit-nodejs-bionic-14x
+FROM socrata/runit-nodejs-bionic-10x
```

Comment:
This is what's been breaking staging Cheetah tests. I'll make a separate issue for the errors around this.